pom.xml

```diff
@@ -77,7 +77,7 @@
     -->
     <javadoc.exec.goal>javadoc</javadoc.exec.goal>
 
-    <slf4jVersion>1.7.7</slf4jVersion>
+    <slf4jVersion>1.7.25</slf4jVersion>
```

Comment:
If we change it, it should be a version in the default core baseline, right? Otherwise we potentially break plugins when they upgrade to the new plugin POM (they will get new API, which is actually not available in their baselines) and there is a miss in the SemVer versioning in slf4j